src/main/java/duke/Task.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import java.time.LocalDateTime;
+
+public class Task {
+
+    public boolean isDone;
+    public String desc;
+    public String type;
```

Comment:
Is it necessary to maintain a type attribute? If it is necessary in your implementation, you might want to consider using `enum` instead of strings as the value of type when you instantiate events/deadlines/... because that will make it more understandable :)